.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!-- Replace ??? with the issue number that this pull request resolves. -->
+Resolves: #???
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds or improves functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] Documentation update
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the [**CONTRIBUTING**](https://github.com/glotzerlab/garnett/blob/master/CONTRIBUTING.md) document.
+- [ ] My code follows the code style of this project.
+- [ ] I have updated the documentation (if relevant).
+- [ ] I have added tests that cover my changes (if relevant).
+- [ ] All new and existing tests passed.
+- [ ] I have updated the [Changelog](https://github.com/glotzerlab/garnett/blob/master/ChangeLog.rst).
```

Comment:
We'll need to be sure this link points to the right place when we fix the changelog.

Reply:
Will keep it in mind. Thanks